coralproject/cay
# Ask 113

## What does this PR do?

The left side bar has been updated.

https://github.com/coralproject/ask/issues/113
## How do I test this PR?
- Check the left sidebar
- You should see two links: Ask and Trust with its icons.

@coralproject/frontend
